src/main/groovy/edu/oregonstate/mist/directoryapi/DirectoryEntityResource.groovy

```diff
@@ -64,6 +69,7 @@ class DirectoryEntityResource extends Resource {
                 )
                 responseBuilder = ok(resultObject)
             } catch (LdapException ldapException) {
+                logger.error("Ldap Exception thrown when getting by search query", ldapException)
```

Comment:
https://github.com/osu-mist/web-api-skeleton/pull/43
Once this PR is thru I plan on merging the generic exception mapper in and leaving the current logging as is.